.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,50 @@
+<!--- STOP! Before you open an issue please search this repository's issues to see if it has already been reported. This helps reduce duplicate issues from being created. -->
+
+**Short description of the issue**:
+
+*description here*
```

Comment:
The filler text between our headers should be comments, so there's no need to delete them